packages/testsuite/cypress/e2e/distributable-web/test-configuration-subsystem-distributable-web-configuration.cy.ts

```diff
@@ -126,12 +126,6 @@ describe("TESTS: Configuration => Subsystem => Distributable Web => Configuratio
     );
   });
 
-  it("Reset", () => {
-    cy.navigateTo(managementEndpoint, "distributable-web");
-    cy.get("#dw-configuration-item").click();
-    cy.resetForm(configurationFormId, managementEndpoint, ["subsystem", "distributable-web"]);
```

Comment:
Alternatively this line can be written to expect current warning, but don't know how to write that.

Reply:
Let's remove the test for now. We'll update this method in scope of https://github.com/hal/berg/issues/44.